tools/pypsa/features.yaml

```diff
@@ -130,103 +196,127 @@ features:
+        - https://docs.pypsa.org/latest/api/networks/network/
     standardised:
       source: []
       value: n
   interface__output:
     gui:
-      source: []
-      value: n
+      source:
+        - https://docs.pypsa.org/latest/user-guide/plotting/charts/#interactive-plots
+        - https://github.com/PyPSA/pypsa-server
+        - https://github.com/PriyeshGosai/pypsa_network_viewer
+        - https://www.bayesian.energy/convexity
+      value: y
     logs:
-      source: []
-      value: n
+      source:
+        - https://docs.pypsa.org/latest/user-guide/logging/
+      value: y
     reports:
-      source: []
-      value: n
+      source:
+        - https://docs.pypsa.org/latest/user-guide/import-export/#excel
+        - https://docs.pypsa.org/latest/user-guide/import-export/#csv-files
+      value: y
     standardised:
       source: []
       value: n
     tables:
       source: []
-      value: n
+      value: y
   optimisation_problem:
     configuration:
-      source: []
-      value: n
+      source:
+        - https://docs.pypsa.org/latest/api/networks/optimize/#pypsa.Network.optimize.solve_model
+      value: y
     decomposition:
       source: []
       value: n
     near_optimal:
-      source: []
-      value: n
+      source:
+        - https://docs.pypsa.org/latest/examples/mga/
+        - https://docs.pypsa.org/latest/api/networks/optimize/#pypsa.Network.optimize.solve_model
+      value: y
     sim + opt:
       source: []
       value: n
     user_defined_constraints:
       value: y
       source:
-        - https://pypsa.readthedocs.io/en/latest/user-guide/optimal-power-flow.html#custom-constraints
+        - https://docs.pypsa.org/latest/user-guide/optimization/custom-constraints/#custom-constraints
     user_defined_variables:
       value: y
       source:
-        - https://pypsa.readthedocs.io/en/latest/user-guide/optimal-power-flow.html#custom-constraints
+        - https://docs.pypsa.org/latest/user-guide/optimization/custom-constraints/#custom-constraints
   postprocessing:
     aggregation:
-      source: []
-      value: n
+      source:
+        - https://docs.pypsa.org/latest/user-guide/statistics/
+      value: y
     curtailment:
       value: y
       source:
-        - https://pypsa.readthedocs.io/en/latest/api/_source/pypsa.statistics.StatisticsAccessor.curtailment.html
+        - https://docs.pypsa.org/latest/api/networks/statistics/#pypsa.statistics.StatisticsAccessor.curtailment
+        - https://docs.pypsa.org/latest/examples/scigrid-lopf-then-pf/#curtailment
     financial:
-      source: []
-      value: n
+      source:
+        - https://docs.pypsa.org/latest/user-guide/statistics/
+      value: y
     future_cost_function:
       source: []
       value: n
     gtc:
-      source: []
-      value: n
+      source:
+        - https://docs.pypsa.org/latest/user-guide/components/lines/
+        - https://docs.pypsa.org/latest/user-guide/components/links/
+      value: y
```

Comment:
As a postprocessing step, this is meant as being able to get GTCs from the tool having input lines and their ratings. It's a hangover from us including power flow models in the same taxonomy. However, am I right in saying this is a possible _preprocessing_ step in pypsa? To get a busmap and convert it to zonal grid transfer capacities? Or is that just a PyPSA-Eur functionality?

Reply:
No, I don't think we have that as standard functionality in PyPSA. ( --> adjusted)